recipes/example/meta.yaml

```diff
@@ -1,72 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
You shouldn't delete the example in your PR; otherwise, when this gets merged nobody else will be able to see the example. :)